### ISSUE TITLE
Creating environment with .yml

### ISSUE THREAD

**Issue by sgaen** [OpenDrift/opendrift#778]
Hello. I have an issue trying to create the environment for Opendrift on windows 10. There is no way to install Imagemagick so I can't go further than this step. Could you help?
[issue.txt](https://github.com/OpenDrift/opendrift/files/7593458/issue.txt)
.

**Comment by gauteh** [comment]
I dont think imagemagick is a hard dependency, maybe it should be removed
from environment.yml.

ons. 24. nov. 2021, 08:24 skrev sgaen ***@***.***>:

**Comment by knutfrode** [comment]
Yes, ImageMagick was added to environment.yml 19 days  ago to get rid of warning that "pillow is used since ImageMagick is not available". But otherwise I think there are no improvements.
So I guess we can simply remove ImageMagick again. But this might perhaps also affect the recent work in #775 ?

**Comment by gauteh** [comment]
Yes, I think we should make a "opendrift performance" page on
opendrift.github.io about various ways to make opendrift faster. The work
done in #775 will also probably have to go in there since we cannot force a
backend on people, but can suggest to use it for faster animation saving.

ons. 24. nov. 2021 kl. 09:01 skrev Knut-Frode Dagestad <
***@***.***>:

**Comment by knutfrode** [comment]
There is already a "performance tips" page, but well hidden in sub-menus (`Theory -> Improving performance`)
https://opendrift.github.io/theory/performance_tips.html

**Comment by manuelaghito** [comment]
? I don't think saving animated gif works without ImageMagick, at least on linux. Or?

**Comment by gauteh** [comment]
Other formats probably work, maybe gif too, with other backends. E.g. ffmpeg.

ons. 24. nov. 2021 kl. 09:10 skrev manuelaghito ***@***.***>:

**Comment by knutfrode** [comment]
Actually not sure. I normally always export to mp4, and then use this utility to create gifs with minimum filesize and maximum quality: https://github.com/OpenDrift/opendrift/blob/master/opendrift/scripts/mp4_to_gif.bash

**Comment by gauteh** [comment]
ons. 24. nov. 2021 kl. 09:07 skrev Knut-Frode Dagestad <
***@***.***>:

> There is already a "performance tips" page, but well hidden in sub-menus (Theory
> -> Improving performance)
> https://opendrift.github.io/theory/performance_tips.html
>

Great, let's put it on the main menu! I am guessing that @manuelaghito's
work have to go there as well in order to keep opendrift generic.

**Comment by knutfrode** [comment]
Yes, good.
Will you remove ImageMagick from environment.yml, or might this interfer with the present work of @manuelaghito ?

**Comment by gauteh** [comment]
Yes, added a PR. It might interfere, but it shouldn't create any
conflicts. @manuelaghito can rebase once it is merged and see if it
has any effect.

ons. 24. nov. 2021 kl. 09:24 skrev Knut-Frode Dagestad
***@***.***>:

**Comment by manuelaghito** [comment]
ok I will try... I guess OpenDrift and gif works also without imagemagick since @knutfrode sais that it was added a few weeks ago.... the gallery always worked with no problems :-) so I guess that even if **anim.save(filename, fps=fps, writer='imagemagick')** was used in _save_animation() it just used pillow instead

**Comment by knutfrode** [comment]
@sgaen Imagemagick is now removed from the dependencies, so hopefully it should work if you update your local code before installing again.

**Comment by gauteh** [comment]
Closing.
